digital/README.md

```diff
@@ -7,13 +7,18 @@
   </div>
 </heading>
 
+<div class="identity-section section1-padding">
+  <h2 class="text-center">What is Digital Delivery?</h2>
```

Comment:
If you put "text-center" on an outer div you can avoid repeating it in both the `<h2>` and `<p>` tags.